ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -51,18 +55,27 @@ Licensed to the Apache Software Foundation (ASF) under one
   private final static Logger LOG = LoggerFactory.getLogger(LoadTable.class);
   //  private final Helper helper;
   private final Context context;
+  private final ReplLogger replLogger;
   private final TableContext tableContext;
   private final TaskTracker tracker;
   private final TableEvent event;
 
-  public LoadTable(TableEvent event, Context context, TableContext tableContext, TaskTracker limiter)
+  public LoadTable(TableEvent event, Context context, ReplLogger replLogger,
+                   TableContext tableContext, TaskTracker limiter)
       throws SemanticException, IOException {
     this.event = event;
     this.context = context;
+    this.replLogger = replLogger;
     this.tableContext = tableContext;
     this.tracker = new TaskTracker(limiter);
   }
 
+  private void createTableReplLogTask(String tableName, TableType tableType) throws SemanticException {
+    ReplStateLogWork replLogWork = new ReplStateLogWork(replLogger,tableName, tableType);
+    Task<ReplStateLogWork> replLogTask = TaskFactory.get(replLogWork, context.hiveConf);
+    ReplLoadTask.dependency(tracker.tasks(), replLogTask);
```

Comment:
tacker is not taking into account this new task being added.

Reply:
I think, it is ok, because this is the last task being added in the current table's DAG. When the caller updates this to loadTaskTracker during post-processing, it will be accounted.